src/Component/People.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import { Component } from "react";
+
+class People extends Component{
+    constructor(){
+        super()
+        this.state = {
+            people: []
+        }
+    }
+fetchPeople = () => {
+    fetch("https://ghibliapi.herokuapp.com/people")
+        .then((res)=> res.json())
+        .then((data)=>{
+            this.setState({
+                people: data,
+            })
+        })
+}
```

Comment:
```suggestion
fetchPeople = () => {
    fetch("https://ghibliapi.herokuapp.com/people")
        .then((res)=> res.json())
        .then((data)=>{
            this.setState({
                people: data,
            })
        })
}

componentDidMount = () => {
     this.fetchPeople()
}
```

Remember, `fetchPeople` won't actually run until you **call** it somewhere. In this case, you want all of your data to be fetched immediately, so you should call it inside of `componentDidMount`.